apps/docs/src/.vitepress/components/IconLibraryItem.vue

```diff
@@ -0,0 +1,75 @@
+<script lang="ts" setup>
+import OnyxIcon from "~components/OnyxIcon/OnyxIcon.vue";
+import type { EnrichedIcon } from "../utils-icons";
+import { ref } from "vue";
+
+const props = defineProps<{
+  icon: EnrichedIcon;
+}>();
+
+const isCopied = ref(false);
+
+const handleCopy = async () => {
+  const { importName, iconName } = props.icon;
+  await navigator.clipboard.writeText(
+    `import ${importName} from "@sit-onyx/icons/${iconName}.svg?raw";`,
+  );
+  isCopied.value = true;
+  setTimeout(() => (isCopied.value = false), 3000);
+};
+</script>
+
+<template>
+  <div class="icon" tabindex="0" @click="handleCopy" @keyup.enter="handleCopy">
+    <OnyxIcon
+      :icon="props.icon.content"
+      :color="props.icon.metadata.deprecated ? 'secondary' : 'currentColor'"
+    />
+    <span class="icon__tooltip">
+      {{ props.icon.tooltipName }}
+      <div v-if="isCopied">Import copy successful!</div>
+    </span>
+  </div>
+</template>
+
+<style lang="scss" scoped>
+.icon {
+  height: 3.5rem;
+  width: 3.5rem;
+  padding: var(--onyx-spacing-md);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  cursor: pointer;
+  border: 0.0625rem solid transparent;
```

Comment:
```suggestion
  border: 1px solid transparent;
```

I think we should make an exception for 1px borders, what do you think?

Reply:
Same for the border in &:hover below